lib/quadquizaminos_web/templates/layout/navbar.html.leex

```diff
@@ -17,6 +17,9 @@
         <%= if Quadquizaminos.Accounts.user_has_role?(@current_user, ["admin"]) do %>
           <li><%= admin_dropdown() %></li>
         <% end %>
+        <% else %>
+         <%= hide_or_show_sign_up_button() %>
+
```

Comment:
The admin button is after the logout button, the order should be `admim` `logout` button not `logout` `admin` @emmyachieng you can move this `<%= link "Logout", to: Routes.auth_path(QuadquizaminosWeb.Endpoint, :delete), method: "delete", class: "button logout"%>` to line 19  after the end tag

Reply:
okay